src/facade/dragonfly_connection.cc

```diff
@@ -596,79 +697,102 @@ auto Connection::IoLoop(util::FiberSocketBase* peer) -> variant<error_code, Pars
   return parse_status;
 }
 
+struct Connection::DispatchOperations {
+  DispatchOperations(SinkReplyBuilder* b, Connection* me)
+      : stats{me->service_->GetThreadLocalConnectionStats()}, builder{b},
+        empty{me->dispatch_q_.empty()}, self(me) {
+  }
+
+  void operator()(AsyncMsg msg);
```

Comment:
all them can be const ref imho.

Reply:
change to ref - const cannot be used because it does change the object there - in each of them differently